namerd/iface/mesh/src/main/scala/io/buoyant/namerd/iface/mesh/ResolverService.scala

```diff
@@ -67,7 +73,7 @@ object ResolverService {
 
     case Addr.Bound(addrs, meta) =>
       val paddrs = addrs.collect(_collectToEndpoint).toSeq
-      mesh.Replicas.OneofResult.Bound(mesh.Replicas.Bound(paddrs))
+      mesh.Replicas.OneofResult.Bound(mesh.Replicas.Bound(paddrs, meta.mapValues(v => mapper.writeValueAsString(v))))
```

Comment:
Since we don't plan on serializing json here, we should just be able to use `toString`.